client/src/pages/CardInfo-components/CardInfo.js

```diff
@@ -0,0 +1,184 @@
+            In List "Math"
+          </Typography>
+          <Button className={classes.closeCard} onClick={() => closeCardInfo()}>
+            &times;
+          </Button>
+        </CardContent>
+        <CardContent className={classes.cardContent}>
+          <Box m={0} className={`${classes.cardBody} ${classes.left}`}>
+            <CardInfoDescription
+              saveDescription={handleSaveDescription}
+              showDescription={showDescription}
+              deleteDescription={handleDeleteDescription}
+            />
+            <CardInfoDeadline
+              saveDeadline={handleSaveDeadline}
+              showDeadline={showDeadline}
+              deleteDeadline={handleDeleteDeadline}
+            />
+            <CardInfoComment
+              saveComment={handleSaveComment}
+              showComment={showComment}
+              deleteComment={handleDeleteComment}
+            />
+          </Box>
+          <Box m={0} className={`${classes.cardBody} ${classes.right}`}>
+            <Box mt={1.5}>
+              <Typography variant="subtitle2" align="center">
+                ADD TO CARD:
+              </Typography>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${
+                  !showDescription ? classes.dBlock : classes.dNone
+                }`}
+                onClick={addDescription}
+              >
+                Description
+              </Button>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${
+                  !showDeadline ? classes.dBlock : classes.dNone
+                }`}
+                onClick={addDeadline}
+              >
+                Deadline
+              </Button>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${
+                  !showComment ? classes.dBlock : classes.dNone
+                }`}
+                onClick={addComment}
+              >
+                Comment
+              </Button>
+            </Box>
+            <Box mt={4}>
+              <Typography variant="subtitle2" align="center">
+                ACTIONS:
+              </Typography>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${classes.dBlock}`}
+              >
+                Move
+              </Button>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${classes.dBlock}`}
+              >
+                Copy
+              </Button>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${classes.dBlock}`}
+              >
+                Share
+              </Button>
+              <Button
+                variant="contained"
+                className={`${classes.add} ${classes.dBlock}`}
+              >
+                Delete
+              </Button>
```

Comment:
nitpick: You can shorten this with a .map()

Reply:
good call!